MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```diff
@@ -199,7 +199,8 @@ private void DisposeConnection(Tuple<IWebSocketConnection, CancellationTokenSour
         {
             Logger.LogDebug("{1} stop transmitting over websocket to {0}", connection.Item1.RemoteEndPoint, GetType().Name);
 
-            connection.Item1.Dispose();
+            // TODO disposing the connection seems to break websockets in subtle ways, so what is the purpose of this function really...
```

Comment:
won't websockets leak without disposing them?..

Reply:
Netstat seemed fine. Dunno if websockets are different from Unix sockets. But disposing breaks things and the old implementation didn't dispose them either